### PR TITLE
Migrate slogan of the day plugin

### PR DESCRIPTION
**Pull request by philipgatzka**

### CODE REVIEW

File: exampleplugins/5.2/SwagSloganOfTheDay/Components/SloganPrinter.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SwagSloganOfTheDay\Components;
+
+class SloganPrinter {
```
Comment (mitelg) [inline exampleplugins/5.2/SwagSloganOfTheDay/Components/SloganPrinter.php]: code style

File: exampleplugins/5.2/SwagSloganOfTheDay/Subscriber/RouteSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    public function onPostDispatch(\Enlight_Controller_ActionEventArgs $args)
+    {
+        /** @var \Enlight_Controller_Action $controller */
+        $controller = $args->get('subject');
+        $view = $controller->View();
+
+        $view->addTemplateDir($this->pluginDirectory . '/Resources/views');
+
+        $view->assign('fontSize', $this->config['fontSize']);
```
Comment (mitelg) [inline exampleplugins/5.2/SwagSloganOfTheDay/Subscriber/RouteSubscriber.php]: please prefix the smarty assignments. fontSize, italic and slogan are too generic

File: exampleplugins/5.2/SwagSloganOfTheDay/Components/SloganPrinter.php (hunk 1 above)
Comment (philipgatzka) [reply exampleplugins/5.2/SwagSloganOfTheDay/Components/SloganPrinter.php]: Done

File: exampleplugins/5.2/SwagSloganOfTheDay/Subscriber/RouteSubscriber.php (hunk 2 above)
Comment (philipgatzka) [reply exampleplugins/5.2/SwagSloganOfTheDay/Subscriber/RouteSubscriber.php]: Done

**Pull request merged**